app/models/account.rb

```diff
@@ -544,7 +544,7 @@ def on_trial?
   # Grabs the support_email if defined, otherwise falls back to the email of first admin. Dog.
   def support_email
     se = self[:support_email]
-    se.presence || admins.first&.email
+    se.presence || first_admin&.email
```

Comment:
There are many occurrences of `admins.first`, should we change all of them?

Reply:
Needs to be evaluated. Maybe but I'm not sure it can be a simple `sed` command because it involves caching.

I would rather leave this for further PRs. This is big enough already.